deploy.sh

```diff
@@ -83,10 +80,11 @@ APX_REGISTRY_PRICE_MANAGER=${APX_REGISTRY_PRICE_MANAGER}
 FEE_MANAGER=${FEE_MANAGER}
 FEE_MANAGER_OWNER=${FEE_MANAGER_OWNER}
 FEE_MANAGER_TREASURY=${FEE_MANAGER_TREASURY}
+PAYOUT_MANAGER=${PAYOUT_MANAGER}
```

Comment:
We should also deploy MerkleTreePathValidator as it is a constructor dependency for PayoutManager (and we may need it elsewhere in the future).